Lets Do This/Controllers/Campaign/LDTCampaignDetailViewController.m

```diff
@@ -209,6 +209,15 @@ - (void)configureReportbackItemCell:(LDTCampaignDetailReportbackItemCell *)repor
     reportbackItemDetailView.userDisplayNameButtonTitle = reportbackItem.user.displayName;
 }
 
+- (CGSize)reportbackItemCellSizeForIndexPath:(NSIndexPath *)indexPath {
+    UINib *reportbackItemCellNib = [UINib nibWithNibName:@"LDTCampaignDetailReportbackItemCell" bundle:nil];
+    LDTCampaignDetailReportbackItemCell *sizingCell = [[reportbackItemCellNib instantiateWithOwner:nil options:nil] firstObject];
+    [self configureReportbackItemCell:sizingCell forIndexPath:indexPath];
+    CGFloat width = CGRectGetWidth([UIScreen mainScreen].bounds);
+    CGSize targetSize = CGSizeMake(width, 0);
```

Comment:
The SO thread suggests a height of 0, yet the [source code the author links to uses a large height](https://github.com/algal/UICVVariableHeightiOS8/blob/2a1924a5c204ecb2418700b75d44220519575a9b/UICVVariableHeightiOS8/MyCell.swift#L59) here suggests using an "excessive height" of 30000. I've tried this and it doesn't change the behavior.